clients/include/testing_gemm_strided_batched_ex.hpp

```diff
@@ -319,7 +319,9 @@ void testing_gemm_strided_batched_ex(const Arguments& arg)
         h_alpha_Tc = float_to_half(arg.alpha);
         h_beta_Tc  = nantest ? 0 : float_to_half(arg.beta);
     }
-    else if(std::is_same<Tc, float>{} || std::is_same<Tc, double>{} || std::is_same<Tc, int32_t>{})
+    else if(std::is_same<Tc, float>{} || std::is_same<Tc, double>{} || std::is_same<Tc, int32_t>{}
+            || std::is_same<Tc, rocblas_float_complex>{}
+            || std::is_same<Tc, rocblas_double_complex>{})
```

Comment:
`is_complex<Tc>` can be used instead of `std::is_same<Tc, rocblas_float_complex>{} || std::is_same<Tc, rocblas_double_complex>{}`

Reply:
Understand. Thanks for catch that. 👍